### PR TITLE
Sj feature profile markup

### PR DESCRIPTION
**Pull request by johnstonstu**
did markup for profile page, waiting on login / user auth to display user data

### CODE REVIEW

File: js/screens/Profile/styles.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    fontWeight: 'bold',
+  },
+  editText: {
+    marginTop: 5,
+    fontFamily: fonts.main,
+  },
+  icon: {
+    height: 60,
+    width: 60,
+    overflow: 'visible',
```
Comment (siddharthparmar7) [inline js/screens/Profile/styles.js]: Please use flexbox instead and Dimensions instead of hard-coded height and width.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work.

your profile screen does not match designer's vision. Please have a profile pic in a circle, the header should be white, not gray, and there are margin padding issues with the icons.

Please make the requested changes.

File: js/screens/Profile/styles.js (hunk 1 above)
Comment (johnstonstu) [reply js/screens/Profile/styles.js]: The icons disappear when set with flex.

Comment (johnstonstu) [reply js/screens/Profile/styles.js]: And the image is a placeholder approved by the designers, the image will appear when the user account info is available.

**Review by siddharthparmar7 — COMMENTED** [review]
Nice work. Your branch has merge conflicts. Please resolve them.

**Review by siddharthparmar7 — APPROVED** [review]
Thank you!

**Pull request merged**